membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -286,14 +286,17 @@ class AccountController(commonActions: CommonActions, override val controllerCom
   case class FilterByProductType(productType: String) extends OptionalSubscriptionsFilter
   case object NoFilter extends OptionalSubscriptionsFilter
 
-  private def productIsInstanceOfProductType(product: Product, productType: String) = product match {
-    // this ordering prevents Weekly subs from coming back when Paper is requested (which is different from the type hierarchy where Weekly extends Paper)
-    case _: Product.Weekly => productType == "Weekly"
-    case _: Product.Paper => productType == "Paper"
-    case _: Product.Contribution => productType == "Contribution"
-    case _: Product.Membership => productType == "Membership"
-    case _: Product.ZDigipack => productType == "Digipack"
-    case _ => productType == product.name // fallback
+  private def productIsInstanceOfProductType(product: Product, productType: String) = {
+    val requestedProductTypeIsSoCalledSubscription: Boolean = productType == "SoCalledSubscription"
```

Comment:
I find this name a bit weird, but I understand the problem you're trying to avoid! Would guardianSubscription be clearer?

Reply:
agree, SoCalled is not specific enough.
it's sometimes important to distinguish between zuora subscriptions (i.e. anything billed recurringly) and guardian subscriptions (anything you get regular content) and it can get confusing where they don't really map well.
From the outside of an API we provide, a subscription should really be a guardian one, however we probably already leak quite a bit of zuora terminology through the layers.  It doesn't hurt to be explicit, but without being confusing.

Reply:
renamed to `ContentSubscription`